frontend/packages/text-editor/src/TextList.test.tsx

```diff
@@ -59,44 +59,54 @@ const renderTextList = (props: Partial<TextListProps> = {}) => {
 };
 
 describe('TextList', () => {
-  test('updateEntryId should be called when id has been changed', async () => {
+  it('should call updateEntryId when the ID is changed in the edit mode', async () => {
     const user = userEvent.setup();
     const updateEntryId = jest.fn();
     const { rerender, initPros } = renderTextList({ updateEntryId });
     rerender(<TextList {...initPros} />);
+
     const toggleEditButton = screen.getAllByRole('button', { name: 'toggle-textkey-edit' });
     await act(() => user.click(toggleEditButton[0]));
-    const idInputs = screen.getAllByRole('textbox', {
-      name: 'tekst key edit',
-    });
-    await act(() => user.dblClick(idInputs[0]));
+    const idInput = screen.getByRole('textbox', { name: 'text key edit' });
+
+    await act(() => user.dblClick(idInput));
```

Comment:
Do we need to use the `dblClick` here? 🤔 
```suggestion
    await act(() => user.click(idInput));
```

Reply:
It's to mark the text in input so the next act can overwrite it. Also possible to replace it with `await act(() => user.clear(idInput));`
What do you think? :)

Reply:
ah, thanks for sharing! Keep it as it is! 😊